v2/robotmk/src/bin/agent.rs

```diff
@@ -51,13 +52,14 @@ fn report_config_content(content: String) {
         config_file_content: content,
     })
     .expect("Unexpected serialization error: ConfigFileContent");
+    println!("<<<robotmk_config:sep(0)>>>");
```

Comment:
There should be different headers for different JSON schemas. Moreover, we should simplify the implementations here, so that the sections are always printed.

If we do not adjust these sections in tandem with commits on the Checkmk side, then it is  very hard to judge whether these changes simplify parsing.

Reply:
As discussed: I unified the configuration into one section.